bin/key_server.js

```diff
@@ -56,7 +56,7 @@ function main() {
           .done(
             function (db) {
               database = db
-              customs = new Customs(config.customsUrl)
+              customs = new Customs(config.customsUrl, database)
```

Comment:
I'm not a big fan of passing the db over to the customs code to it to call. Seems like action at a distance once it becomes a little spread out rather than all db functions being called in this file. (See next comment too.)